frontend/src/scenes/billing/v2/test/Billing.tsx

```diff
@@ -50,6 +42,26 @@ export function BillingV2({ redirectPath = '' }: BillingV2Props): JSX.Element {
         return <SpinnerOverlay />
     }
 
+    if (!billing && !billingLoading) {
```

Comment:
I cleaned up a bit the ternary galore as my eyes were criss-crossing, so I moved this outside of the main return, and removed a few ? : ternaries